<launch>
    <node type="lvio_2d_node" pkg="lvio_2d" name="lvio_2d_node"  output="screen" >
        <rosparam file="$(find lvio_2d)/config/market.yaml" command="load"/> 
    </node>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lvio_2d)/launch/display.rviz" />
</launch>